test/fixtures/implicit-returns/loop.js

```diff
@@ -1,11 +1,11 @@
 fn(x) ->
-  for i from 0 til 10:
+  for let i=0; i<10; i++:
```

Comment:
let's move to the higher-level `for idx i in Array(10):` form in these higher-level tests?